Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -227,21 +227,31 @@ private Vector3 GetPointersCentroid()
         private Vector3 GetPointersVelocity()
         {
             Vector3 sum = Vector3.zero;
+            int numControllers = 0;
             foreach (var p in pointerIdToPointerMap.Values)
             {
-                sum += p.Controller.Velocity;
+                if (p.Controller != null)
```

Comment:
Would be nice to explain why we are doing this null check here instead of just using pointerIdToPointerMap.Count, it's not obvious to me :-)

Reply:
> Not to sound self-referential but this PR fixes the issue it fixes, i.e.: #4117. I agree ManipulationHandler could do with a general review (e.g. use of controller velocities) but in the meantime fixing those null references does no harm and help us with upcoming changes to gaze pointer.

I think Stephen actually has a point. This will unblock people yes, but really we should remove all that velocity code from ManipulationHandler. I created a separate bug to track that. It sounds like this at least temporarily unblocks you though, which is why I approved this PR.